fixtures/etc/source-four-led-series-3-daylight-hdr.json

```
@@ -0,0 +1,358 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Source Four LED Series 3 Daylight HDR",
+  "shortName": "EtcSF3DaylightHDR",
+  "categories": ["Dimmer", "Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Jeppe Cohrt", "Dyami Caliri"],
+    "createDate": "2019-07-11",
+    "lastModifyDate": "2022-04-12"
```

Change date to today (merge date):

```suggestion
    "createDate": "2022-05-07",
    "lastModifyDate": "2022-05-07"
```